testify/test_runner.py

```diff
@@ -212,9 +207,22 @@ def get_tests_for_suite(self, selected_suite_name):
                 if not selected_suite_name or TestCase.in_suite(test_method, selected_suite_name):
                     yield test_method
 
-    def list_tests(self, selected_suite_name=None):
+    def list_tests(self, format, selected_suite_name=None):
         """Lists all tests, optionally scoped to a single suite."""
         for test in self.get_tests_for_suite(selected_suite_name):
-            print(self.get_test_method_name(test))
+            name = self.get_test_method_name(test)
+            if format == 'txt':
+                print(name)
+            elif format == 'json':
+                testcase = test.__self__
+                print(json.dumps(
+                    dict(
+                        test=name,
+                        suites=sorted(testcase.suites(test)),
```

Comment:
Can the method itself have suites?  The `@T.suite` decorator can apply to it, but does it do anything?  Should it raise maybe if it doesn't?